<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cotaButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>1 - Cotação</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fechamentoButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>2 - Fechamento</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cotallButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>3 - Cotação de Todas Moedas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
